regression/tests/whitelabel/course_enrollment_test.py

```diff
@@ -201,3 +202,31 @@ def assert_enrollment_and_unenroll(self):
         """
         self.assert_course_added_to_dashboard()
         self.unenroll_using_ui()
+
+    def pay_with_cybersource(self):
+        """
+        Make payment using cybersource.
+        """
+        # Fill out all the billing and payment details and submit the form
+        self.otto_payment_using_cyber_source()
+        # Application should take user to the receipt page
+        # Verify on receipt page that information like course title,
+        # course price, total price order date and billing to is
+        # displayed correctly
+        self.verify_receipt_info()
+        self.receipt_page.click_in_nav_to_go_to_dashboard()
+
+    def verify_receipt_info(self):
+        """
+        Verify that Course title, Course Price, total price and order date on
+        receipt page is correct.
+        """
+        self.assertIn(self.course_title, self.receipt_page.order_desc)
+        self.assertEqual(
+            unicode(datetime.datetime.utcnow().date()),
```

Comment:
note that there is a slight chance that this will fail if the test execution crosses a boundary of midnight. That said, I think that the risk is so slight that it's fine to leave it like this. May be worth a comment in the code though.